packages/flodash/lib/modules/string/to_upper.dart

```diff
@@ -0,0 +1,4 @@
+///Converts string, as a whole, to upper cas
+String toUpper([String string = '']) {
```

Comment:
Also shall we add in a deprecation warning into this? This functionality is available inbuilt ne. Refer to a few array modules, the have example deprecation warnings